### DISCUSSION TITLE
Help pencerahan sedikit dong yang ngerti

### DISCUSSION THREAD

**Discussion by newbiepuh** [deaafrizal/forum#80] [The Lounge]
jadi gini, aku masih baru dalam hal coding dll. untuk saat ini aku belajar otodidak, cuma bisa copy2 punya orang trus edit2 dikit, ga bisa build up dari 0. nah dalam tahap pembelajaran ini,sebaiknya aku memang harus dari 0, ngerti bahasa program kayak Jscrip, Phyton dll, atau tetap ngelanjutin copy2 punya orang trus edit ? soalnya kalo cuma sekedar copy + edit kayaknya ga ada kemajuan gitu. trus tiap mau nambah bahan kedepannya kayak ngeblank, gatau harus apa. satu sisi pengen maju, tp satu sisi lain kekurangan bahan. maklum, cuma bisa belajar otodidak doang, ga bisa ikut les / private / kuliah. kalo ada saran dll tolong di bantu yakk

**Comment by seporang** [comment]
yoo, bisa aja tetep copy, tapi mending lu pakai bantuan chatgpt/ ai yg responsenya sesuai kita, atau lu belajar chat response, disitu lu juga bisa sambil belajar
